### PR TITLE
Document iMIP API change for 26

### PR DESCRIPTION
**Pull request by miaulalala**
For https://github.com/nextcloud/server/pull/34893

image

### CODE REVIEW

File: developer_manual/digging_deeper/groupware/calendar.rst [hunk 1]
```diff
@@ -161,6 +161,29 @@ Calendars that only return `ICalendar` are implicitly read-only. If your app's c
 
     }
 
+Handling iMIP data 
+~~~~~~~~~~~~~~~~~~
+
+You may implement the ``IHandleIMipMessage`` interface to process iMIP data you receive in a client and want to pass on for processing to the backend. 
+
+Please be aware that there are some security considerations to take into account.
```
Comment (ChristophWurst) [inline developer_manual/digging_deeper/groupware/calendar.rst:169]: Can we also link to some resources about the security considerations? Is this related to the next paragraph with the RFC link? Then remove the separating line breaks.

Comment (miaulalala) [reply developer_manual/digging_deeper/groupware/calendar.rst:169]: Yes it is.

**Review by ChristophWurst — APPROVED** [review]
:rocket:
